Makefile.tools

```diff
@@ -185,6 +188,19 @@ $(GOJSONTOYAML) gojsontoyaml: $(TOOLS_DIR)
 		GOBIN=$(TOOLS_DIR) go install github.com/brancz/gojsontoyaml@latest ;\
 	}
 
+.PHONY: shellcheck
+$(SHELLCHECK) shellcheck: $(TOOLS_DIR)
+	@{ \
+		set -ex ;\
+		[[ -f $(SHELLCHECK) ]] && exit 0 ;\
+		cd $$(mktemp -d) ;\
+		curl -sSLo shellcheck-stable.tar.xz "https://github.com/koalaman/shellcheck/releases/download/stable/shellcheck-stable.linux.x86_64.tar.xz";\
```

Comment:
How about we use the same version as the one declared above to be consistent ? 

`https://github.com/koalaman/shellcheck/releases/download/$(VERSION)/shellcheck-v$(VERSION).linux.x86_64.tar.xz`